README.md

```diff
@@ -8,18 +8,20 @@
 
 ![](img/screenshots/screenshot1.png)
 
-- **📝 Focused writing:** No distractions, only the formatting you need.
+## Features
+
+- **📝Simple Focused writing:** No distractions, only the formatting you need.
```

Comment:
```suggestion
- **📝 Simple focused writing:** No distractions, only the formatting you need.
```

Reply:
Done, Thanks for pointing out

Reply:
Sorry still missing the space compared to the other list entries between the emoji and the text :wink: Otherwise :+1:

Reply:
Thanks , That's embarrassed lol